### ISSUE TITLE
Not an issue, a suggestion

### ISSUE THREAD

**Issue by XH14MZAH3** [jacky8399/PortableBeacons#7]
not an issue but a suggestion, I wonder if you can add a setting so it only affects other people/only affects you? (I don’t know if there is one already but I don’t see one)

**Comment by jacky8399** [comment]
By that do you mean you want an upgrade that allows you to share potion effects? That could certainly be an interesting feature.

**Comment by XH14MZAH3** [comment]
Yes, and it could also be a great PvP method, and if you are looking to add this feature I’m suggesting, you could give it the AOE of a normal beacon, and have an enchantment increase the AOE
